sycl/include/CL/sycl/handler.hpp

```diff
@@ -1714,6 +1716,20 @@ class __SYCL_EXPORT handler {
     }
   }
 
+  /// Fills the specified memory with the specified pattern.
+  ///
+  /// \param Ptr is the pointer to the memory to fill
+  /// \param Pattern is the pattern to fill into the memory.  T should be
+  /// trivially copyable.
```

Comment:
"T should be trivially copyable"
Can we add a static_assert check for that?

Reply:
Yes, we can.